src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -71,6 +71,11 @@ provideEditingConfig(Homepage, {
       description:
         'If your chosen font supports multiple weights, pick the one that best fits your design. Default: 500',
     },
+    siteSinglePage: {
+      title: 'Single page site',
+      description:
+        'If activated, only the selected page will be accessible, even for logged-in users. All other pages will display a 404 not found page.',
+    },
```

Comment:
* [ ] FYI The site settings are becoming more and more crowded. Maybe we should group them into frequently used (e.g. for fresh trials or demos) and separate tabs for infrequently/special purpose.